torch/nn/_functions/packing.py

```diff
@@ -14,7 +14,7 @@ def forward(ctx, input, lengths, batch_first):
 
         steps = []
         batch_sizes = []
-        lengths_iter = reversed(lengths)
+        lengths_iter = reversed(list(lengths))
```

Comment:
Because `lengths` used to be a list but now it is not, it is worth commenting that `lengths` is a `Tensor` here.

Reply:
++